fili-core/src/main/java/com/yahoo/bard/webservice/data/dimension/impl/LuceneSearchProvider.java

```diff
@@ -342,90 +336,26 @@ public void replaceIndex(String newLuceneIndexPathString) {
             String tempDir = oldLuceneIndexPath.resolveSibling(oldLuceneIndexPath.getFileName() + "_old").toString();
 
             LOG.trace("Moving old Lucene index directory from {} to {} ...", luceneIndexPath, tempDir);
-            moveDirEntries(luceneIndexPath, tempDir);
+            Utils.moveDirEntries(luceneIndexPath, tempDir);
```

Comment:
As mentioned in PR description as well as CHANGELOG, `moveDirEntries()`, as an utility method, is moved to `Utils`.